public/search.php

```diff
@@ -37,14 +39,36 @@
         $min_price = number_format($row['minimum_price']);
         $max_price = number_format($row['maximum_price']);
         $search_view = 'includes/search-valid.php';
+        $latitude = $row['latitude'];
+        $longitude = $row['longitude'];
     }
-
     db_query_error($sql_query);
 }
-
-
 ?>
 
-<?php require_once($search_view); ?>
+<div class="search-page-wrapper" id="wrapper">
+    <!-- sidemap -->
+    <div id="sidemap-wrapper">
+        <div class="sidemap-nav" id="sidemap-nav"></div>
+    </div>
+    <!-- /#sidemap-wrapper -->
+    <?php
+    echo "<script>
+            function searchResultMap() {
+                var point = { lat: ".$latitude.", lng: ".$longitude." };
+                var map = new google.maps.Map(document.getElementById('sidemap-nav'), { zoom: 18, center: point });
+                var marker = new google.maps.Marker({
+                position: point,
+                map: map,
+                // icon: ''
+                });
+            }
+        </script>";
+    echo '<script async defer src="https://maps.googleapis.com/maps/api/js?key='.MAP_API_KEY.'&callback=searchResultMap"></script>';
+    
+    ?>
+
+    <?php require_once($search_view); ?>
+    <?php require_once('includes/footer.php'); ?>
```

Comment:
Can we require the footer.php after the div below since the footer contains the closing HTML tag `</html>`?